build.gradle

```diff
@@ -80,7 +81,7 @@ ext {
   targetSdkVersion = 30
   minSdkVersion = 24
 
-  androidxAppCompat = "androidx.appcompat:appcompat:1.2.0"
+  androidxAppCompat = "androidx.appcompat:appcompat:1.3.1"
```

Comment:
This is needed for Compose to work properly.

Reply:
It might be worth doing the upgrade in the dependabot PR as it requires a bunch of lint fixes